src/plugins/lanplus/lanplus.c

```diff
@@ -603,11 +603,11 @@ ipmiv2_lan_ping(struct ipmi_intf * intf)
  * Returns: the ipmi_rs packet describing the/a response we expect.
  */
 static struct ipmi_rs *
-ipmi_lan_poll_single(struct ipmi_intf * intf)
+ipmi_lan_poll_single(struct ipmi_intf *intf)
```

Comment:
I really think that formatting changes must not be confused in the same commit with code changes like removing the unused variables. It's quite hard to review.

Reply:
Yeah, originally I was keeping them quite separate, however you've instructed me to fix every line I touch and then the surrounding lines as well, it just became a bit of a back and forth to have a style fix commit, then a real fix commit, back and forth for a given code block.

I did push a PR that will deal with style that requires your feedback, if that's used, I can fix up all the files and then rebase these patches removing the extra noise.